docs/source/datasets.md

```diff
@@ -0,0 +1,46 @@
+# Datasets
+*flarestack* is designed to work with different types of datasets.
+
+Datasets are stored under the *flarestack* data directory (`$FLARESTACK_DATA_DIR`). Note that this is different from the `flarestack__data` directory that is automatically created under `$FLARESTACK_SCRATCH_DIR`. The former is a static repository of datasets, the latter is the actual working directory of *flarestack*. Python modules acting as interfaces to the stored datasets are included under `flarestack/data`.
```

Comment:
So the principle is that FLARESTACK_DATA_DIR is the input directory for the code, and `flarestack__data` contains 'disposable' output from flarestack. In principle you can delete everything in `flarestack__data`, because the code can rebuild it.

Reply:
> So the principle is that FLARESTACK_DATA_DIR is the input directory for the code, and `flarestack__data` contains 'disposable' output from flarestack. In principle you can delete everything in `flarestack__data`, because the code can rebuild it.

This is clear to me, also not much in contradiction with what I wrote (?). Can you suggest a rewording, if you think it is necessary?

Reply:
Actually, sorry, you're right. It reads fine.